YieldFarm.sol

```diff
@@ -49,7 +49,12 @@ contract YieldFarm {
         address lpTokenAddress,
         address stakeContract,
         address communityVault
-    ) public {
+    ) {
+        require(syloTokenAddress != address(0), "Sylo token address is required");
+        require(lpTokenAddress != address(0), "LP token address is required");
+        require(stakeContract != address(0), "Stake address is required");
+        require(communityVault != address(0), "Community Vault address is required");
```

Comment:
Should this be

```
        require(syloTokenAddress != address(0), "Sylo token address is required");
        require(lpTokenAddress != address(1), "LP token address is required");
        require(stakeContract != address(2), "Stake address is required");
        require(communityVault != address(3), "Community Vault address is required");
```

Reply:
`address(0)` means address 0x0000000000...0000 or the default value for a struct